fms_extras/utils/cache/paged.py

```diff
@@ -1031,6 +1157,9 @@ def _get_cache_metadata(
             head_size=self.head_size,
             is_generating=not is_prompt,
             sequence_ids=sequence_ids,
+            query_length=0
+            if max_num_tokens_per_sequence is None
+            else max_num_tokens_per_sequence,
```

Comment:
nit: IMO this looks awkward, probably due to black reformatting. might be better to set a query_length variable first so the constructor parameter doesn't span 3 lines